x-pack/plugins/reporting/public/lib/stream_handler.ts

```diff
@@ -107,9 +106,9 @@ export class ReportingNotifierStreamHandler {
             _source: { status: jobStatus },
           } = job;
           if (storedJobs.includes(jobId)) {
-            if (jobStatus === JOB_STATUS_COMPLETED || jobStatus === JOB_STATUS_WARNINGS) {
```

Comment:
Cleaned up a duplicated constant here: `JOB_STATUS_COMPLETED` and `JOB_STATUSES.COMPLETED`